packages/next/pages/discussions.js

```diff
@@ -52,11 +52,13 @@ export default withLoginUserRedux(({ posts }) => {
 });
 
 export const getServerSideProps = withLoginUser(async (context) => {
-  const { page, page_size: pageSize } = context.query;
+  const { page, page_size: pageSize, label } = context.query;
 
-  const [{ result: posts }] = await Promise.all([
-    nextApi.fetch(`posts`, { page: page ?? 1, pageSize: pageSize ?? 50 }),
-  ]);
+  let q = { page: page ?? 1, pageSize: pageSize ?? 50 };
+  if (label) {
```

Comment:
We don't need this branch, and we can just put `label` into `q`, because we will check `label` on backend.